internal/migrations/20210713123129_populate_infra_env_test.go

```diff
@@ -50,8 +50,7 @@ var _ = Describe("populateInfraEnv", func() {
 			ID: &hostID,
 			// [TODO] - currently we check migration by adding dummy value and seeing it replaced
 			// in case we add host v1 model to the swagger, then we can replace the models.Host with it
-			InfraEnvID: strfmt.UUID(uuid.New().String()),
-			ClusterID:  &clusterID,
+			ClusterID: &clusterID,
```

Comment:
Why is this changing? What does this have to do with gorm v2?

Reply:
Will revert